google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryProjectState.java

```diff
@@ -134,7 +135,9 @@ void syncManagedProjectLibraries() {
                   MavenDomProjectModel model =
                       MavenDomUtil.getMavenDomProjectModel(project, mavenProject.getFile());
 
-                  return model.getDependencies().getDependencies();
+                  return model != null
+                      ? model.getDependencies().getDependencies()
```

Comment:
will `model.getDependencies()` always be non-null as well?

Reply:
it is @NotNull. But I am still checking in the caller that the return of this method != null.

Reply:
oh nm. yes model.getDependencies is @NotNull. Thought you were asking about getDependencies#getDependencies at first